.github/workflows/publish-release.yml

```diff
@@ -64,4 +64,4 @@ jobs:
                   GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
               run: |
                 gh release create v$VERSION --title "module-ballerina-soap-v$VERSION"
-                gh pr create --base main --title "[Automated] Sync main after $VERSION release" --body "Sync main after $VERSION release"
+                gh pr create --base master --title "[Automated] Sync master after $VERSION release" --body "Sync master after $VERSION release"
```

Comment:
for the 7.x branch we can make it `2201.7.x`

Reply:
Fixed